eessi-2023.06-eb-4.8.1-2022a.yml

```
@@ -5,5 +5,8 @@ easyconfigs:
       options:
         from-pr: 18870
   - foss-2022a
+  - ESPResSo-4.2.1-foss-2022a:
+      options:
+        from-pr: 18963
   - SciPy-bundle-2022.05-foss-2022a
-  - BAMM-2.5.0-foss-2022a.eb
\ No newline at end of file
+  - BAMM-2.5.0-foss-2022a.eb
```

@maxim-masterov Let's try and preserve order in which installations were done here, so:

```suggestion
  - SciPy-bundle-2022.05-foss-2022a
  - BAMM-2.5.0-foss-2022a.eb
  - ESPResSo-4.2.1-foss-2022a:
      options:
        from-pr: 18963